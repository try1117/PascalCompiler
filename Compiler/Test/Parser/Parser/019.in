program test;
var
  a : array[1..10] of integer;

begin
end.
