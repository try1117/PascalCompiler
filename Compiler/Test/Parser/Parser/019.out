test : function()
   resultType : Nil

test declarations:
   a : Array [1, 10] of Integer

